src/client/index.js

```diff
@@ -201,9 +206,16 @@ function mapMethods( methods, clientKey ) {
 	}, {} );
 }
 
+function mapMutations( mutations, methods, endpoints ) {
+	return Object.keys( mutations ).reduce( ( mappedMutations, mutationName ) => {
+		mappedMutations[ mutationName ] = mutations[ mutationName ]( methods, endpoints );
+		return mappedMutations;
+	}, {} );
+}
+
 function mapSelectors( selectors, clientGetData, clientRequireData ) {
 	return Object.keys( selectors ).reduce( ( mappedSelectors, selectorName ) => {
 		mappedSelectors[ selectorName ] = selectors[ selectorName ]( clientGetData, clientRequireData );
 		return mappedSelectors;
-	}, [] );
+	}, {} );
```

Comment:
Curious about this change here - is it just to make it the same as mutations being stored in an object?

Reply:
It was actually incorrect, and I didn't notice it before.